torch/_dynamo/trace_rules.py

```diff
@@ -1183,7 +1185,6 @@
         "torch._C._test_only_populate_upgraders",
         "torch._C._test_only_remove_entry_to_op_version_map",
         "torch._C._test_only_remove_upgraders",
-        "torch._C._to_dlpack",
```

Comment:
oh?

Reply:
Previously, this was allow_in_graph'ed, but would always access FakeTensor.data_ptr and then graph break due to the return being a Dynamo-unsupported type. This PR makes it so that we alays graph break on `torch._C._to_dlpack`. The end result is the same (graph break) but we end up avoiding the FakeTensor.data_ptr access.